### PR TITLE
{bio} [goolf/1.4.10] Bcftools 1.1 new easyconfig (REVIEW)

### PR DESCRIPTION
**Pull request by rjeschmi**
This has a corresponding new easyblock:

https://github.com/hpcugent/easybuild-easyblocks/pull/511

### CODE REVIEW

**Comment by fgeorgatos** [comment]
@boegel: test & merge? is there a cause to block this one?

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/easyconfigs/b/BCFtools/BCFtools-1.1-goolf-1.4.10.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+description = """ SAM Tools provide various utilities for manipulating alignments in the SAM format, 
+ including sorting, merging, indexing and generating alignments in a per-position format."""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+sources = [SOURCELOWER_TAR_BZ2]
+source_urls = [('http://sourceforge.net/projects/samtools/files/samtools/%s' % (version), 'download')]
+
+patches = ['BCFtools_Makefile.patch']
```
Comment (boegel) [inline easybuild/easyconfigs/b/BCFtools/BCFtools-1.1-goolf-1.4.10.eb]: include version in patch filename?

File: easybuild/easyconfigs/b/BCFtools/BCFtools-1.1-goolf-1.4.10.eb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+homepage = 'http://www.htslib.org//'
+description = """ SAM Tools provide various utilities for manipulating alignments in the SAM format, 
+ including sorting, merging, indexing and generating alignments in a per-position format."""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+toolchainopts = {'optarch': True, 'pic': True}
+
+sources = [SOURCELOWER_TAR_BZ2]
+source_urls = [('http://sourceforge.net/projects/samtools/files/samtools/%s' % (version), 'download')]
```
Comment (boegel) [inline easybuild/easyconfigs/b/BCFtools/BCFtools-1.1-goolf-1.4.10.eb]: please use `%(version)s`

**Comment by rjeschmi** [comment]
newer version got merged 

https://github.com/hpcugent/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/b/BCFtools/BCFtools-1.2-foss-2015a.eb

It should be the basis of any backports anyway.

**Pull request closed without merging**